libpod/options.go

```diff
@@ -1453,7 +1453,11 @@ func WithNamedVolumes(volumes []*ContainerNamedVolume) CtrCreateOption {
 		for _, vol := range volumes {
 			mountOpts, err := util.ProcessOptions(vol.Options, false, "")
 			if err != nil {
-				return errors.Wrapf(err, "error processing options for named volume %q mounted at %q", vol.Name, vol.Dest)
+				return errors.Wrapf(err, "processing options for named volume %q mounted at %q", vol.Name, vol.Dest)
+			}
+
+			if err = parse.ValidateVolumeCtrDir(vol.Dest); err != nil {
+				return errors.Wrapf(err, "validating destination path for named volume %q mounted at %q", vol.Name, vol.Dest)
```

Comment:
I think this will still stutter vol.Dest?

Reply:
oh my bad let me fix that i missed it.

Reply:
@rhatdan Fixed but there is more `Error: error` stutter coming from `libpod/runtime_ctr.go` should i fix that in the same PR ?

Reply:
No let's move that to a separate bigger PR.